src/MyTested.WebApi/IServerBuilder.cs

```diff
@@ -39,6 +39,13 @@ public interface IServerBuilder
         IServerBuilder WithDefaultRequestHeaders(IDictionary<string, IEnumerable<string>> headers);
 
         /// <summary>
+        /// Removes a previously added default header from every request tested on the server.
+        /// </summary>
+        /// <param name="name">Name of the header.</param>
+        /// <returns></returns>
+        IServerBuilder RemoveDefaultRequestHeader(string name);
```

Comment:
Let's rename the method to have a more expressive syntax. The library uses `With`, `Without` and `Containing` mostly. I suggest `WithoutDefaultRequestHeader`. If you have another suggestion, please, propose! 😃